Makefile

```diff
@@ -96,3 +99,8 @@ release-publish:
 	./bin/publish-release.sh $(APP_NAME) $(IMAGE_NAME) $(IMAGE_TAG)
 
 release-deploy:
+ifneq ($(filter $(ENV_NAME),$(ENVIRONMENTS)),)
+	./bin/deploy-release.sh $(APP_NAME) $(IMAGE_TAG) $(ENV_NAME)
+else
+	@echo "Please enter: make release-deploy ENV_NAME=<env_name>. The value for env_name must be one of these: $(ENVIRONMENTS)"
+endif
```

Comment:
nit: for input validation, adopt the "if validation fails, return early" pattern.

so instead of 
```
if validation check passes
  do thing
else
  do error handling
```

do this pattern

```
if validation fails
  error handling
  break/exit/return

do thing
```

it makes the flow easier to read.

also, is it possible to indent the if checks?